auth/src/test/java/com/firebase/ui/auth/ui/phone/SpacedEditTextTest.java

```diff
@@ -52,9 +52,9 @@ public void setUp() throws Exception {
         context = mock(Context.class);
         array = mock(TypedArray.class);
 
-        when(array.getFloat(R.styleable.SpacedEditText_spacingProportion, 1)).thenReturn
+        when(array.getFloat(R.styleable.fui_SpacedEditText_fui_spacingProportion, 1)).thenReturn
```

Comment:
I think that's an effect of the build tools appending the two resource names together. I'll make sure it's reduced to `SpacedEditText_fui_spacingProportion`.